op-bindings/bindgen/utils.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type contractsList struct {
+	Local []string `json:"local"`
+}
+
+// readJSONFile reads a JSON file from the given `filePath` and unmarshals
+// its content into the provided result interface. It logs the path of the file
+// being read.
+//
+// Parameters:
+// - logger: An instance of go-ethereum/log
+// - filePath: The path to the JSON file to be read.
+// - result: A pointer to the structure where the JSON data will be unmarshaled.
+//
+// Returns:
+// - An error if reading the file or unmarshaling fails, nil otherwise.
+func readJSONFile(logger log.Logger, filePath string, result interface{}) error {
+	logger.Debug("Reading contract list", "filePath", filePath)
+	contractData, err := os.ReadFile(filePath)
+	if err != nil {
+		return err
+	}
+
+	return json.Unmarshal(contractData, result)
+}
```

Comment:
```suggestion
func readContractsList(logger log.Logger, filePath string) (contractsList, error) {
        var contracts contractsList
	logger.Debug("Reading contract list", "filePath", filePath)
	
	contractData, err := os.ReadFile(filePath)
	if err != nil {
		return contracts, err
	}

	return json.Unmarshal(contracts, result)
}
```

This function is specific to the contracts list (debug log stmt) so better to be opinionated